### DISCUSSION TITLE
Help understanding incomplete status for run

### DISCUSSION THREAD

**Discussion by vasanth-asokan** [langchain-ai/langsmith-sdk#1229] [Q&A]
I have a langsmith trace that shows a streaming OpenAI call (via langchain) get logged in langsmith as incomplete. On my server, the call never completed, there were no timeouts, exceptions as far as we can tell. I do have timeouts set on the langchain call (90 seconds).

Does anyone know what INCOMPLETE means in langsmith parlance? I did a quick search on the langsmith docs and did not see run statuses being cataloged anywhere. I don't see any additional info that's illustrative in the `Metadata` tab in the langsmith UI for this run.

<img width="279" alt="Screenshot 2024-11-19 at 8 46 31 AM" src="https://github.com/user-attachments/assets/904e7cf1-a757-4bb5-93db-79b5e77c94c8">

**Comment by hugobiais** [comment]
Has anyone figured out why this happens?

**Comment by MH4GF** [comment]
I want to know what this status means too!

**Comment by novaware-io** [comment]
This happens to some of my langgraph too, any idea why?

**Comment by mikegreen** [comment]
Had this problem with basic implementation in a Cloudflare Worker... 
In my case, the req would hang out at "Pending" for an hour then "Incomplete" 

Still can't find this in the docs, but appears to be explained/solved 
https://github.com/langchain-ai/langsmith-sdk/issues/1875#issuecomment-3145205406
